draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -333,8 +333,8 @@
       in the BUNDLE group or reject it.
     </t>
     <t>
-      The usage of the 'bundle-only' attribute is only defined for a bundled "m=" line with
-      a zero port value, within an offer. Other usage is unspecified.
+      The usage of the 'bundle-only' attribute is only defined for a bundled "m=" section with
+      a zero port value on its "m=" line, within an offer. Other usage is unspecified.

```

Comment:
Would we need to say "on its "m=" line", or could we simply say '"m=" section with a zero port value'?